lib/app/service/test/builder.go

```diff
@@ -57,6 +57,15 @@ func CreateDummyApplication2(apps app.Applications, loc loc.Locator, dependencie
 	return createApp(loc, dependencies, apps, c)
 }
 
+// CreateDummyApplicationInMultipleServices creates a test application with a valid manifest
+// in all of the specified application services
+func CreateDummyApplicationInMultipleServices(loc loc.Locator, c *C, services ...app.Applications) {
```

Comment:
Nit, I wonder why not just tweak `CreateDummyApplication` to accept variadic argument. This would require reordering its arguments though so "apps" goes last, is that a lot of refactoring?